tests/utils/collections/policyTags.ts

```diff
@@ -4,7 +4,8 @@ import type {PolicyTagLists, PolicyTags} from '@src/types/onyx';
 export default function createRandomPolicyTags(tagListName: string, numberOfTags = 0): PolicyTagLists {
     const tags: PolicyTags = {};
     for (let i = 0; i < numberOfTags; i++) {
-        const tagName = randWord();
+        // prevent the tagName can be duplicated
```

Comment:
You might need to split this suggestion onto multiple lines. My suggestion improves the comment by explaining _why_ the duplicates need to be prevented.

```suggestion
        // Prevent the tag name from being duplicated, which can happen when a lot of tests are being ran and can cause tests to fail because tag lists must always contain a unique set of tags
```